listenbrainz/webserver/errors.py

```diff
@@ -166,7 +167,9 @@ def service_unavailable(error):
 
     @app.errorhandler(APIError)
     def api_error(error):
-        return jsonify(error.to_dict()), error.status_code
+        resp = make_response(jsonify(error.to_dict()))
+        resp.headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
the cross-domain decorator should already handle this right? https://github.com/metabrainz/listenbrainz-server/blob/2d47f7b4/listenbrainz/webserver/decorators.py#L7

Any way that we can go through that code path here also?

That way we'll have a single implementation of these headers, leading to less chances of security issues. It also gives us the allowed headers header too.

Reply:
Done